jpsonic-main/src/main/java/com/tesshu/jpsonic/service/upnp/ApacheUpnpServiceConfiguration.java

```diff
@@ -40,17 +30,17 @@
  */
 public class ApacheUpnpServiceConfiguration extends DefaultUpnpServiceConfiguration {
 
-    @SuppressWarnings("rawtypes")
-    @Override
-    public StreamClient createStreamClient() {
-        return new StreamClientImpl(new StreamClientConfigurationImpl(Executors.newCachedThreadPool()));
-    }
-
-    @SuppressWarnings("rawtypes")
-    @Override
-    public StreamServer createStreamServer(NetworkAddressFactory networkAddressFactory) {
-        int listenPort = SettingsService.getDefaultUPnPPort();
-        return new StreamServerImpl(new StreamServerConfigurationImpl(
-                listenPort < 1 ? networkAddressFactory.getStreamListenPort() : listenPort));
-    }
+    // @SuppressWarnings("rawtypes")
+    // @Override
+    // public StreamClient createStreamClient() {
+    // return new StreamClientImpl(new StreamClientConfigurationImpl(Executors.newCachedThreadPool()));
+    // }
+    //
+    // @SuppressWarnings("rawtypes")
+    // @Override
+    // public StreamServer createStreamServer(NetworkAddressFactory networkAddressFactory) {
+    // int listenPort = SettingsService.getDefaultUPnPPort();
+    // return new StreamServerImpl(new StreamServerConfigurationImpl(
+    // listenPort < 1 ? networkAddressFactory.getStreamListenPort() : listenPort));
+    // }
 }
```

Comment:
It will be commented out. (Cannot be compiled). Jpsonic cannot handle this, so a patch version of the 3rd party library will be created.

Reply:
Already listed as `Non-Goal`. This issue will be resolved in a subsequent pull request.